central/complianceoperator/v2/integration/service/convert.go

```diff
@@ -28,18 +33,23 @@ func convertStorageIntegrationToV2(ctx context.Context, integration *complianceD
 	}
 
 	opStatus := v2.COStatus_UNHEALTHY
-	if integration.OperatorStatus == storage.COStatus_HEALTHY {
+	if integration.OperatorStatus != nil && *integration.OperatorStatus == storage.COStatus_HEALTHY {
```

Comment:
Would it be easier to add `GetOperatorStatus()` and similar functions for pointer fields in `IntegrationDetails` struct?

Reply:
great idea.  Thanks